frame-metadata/src/lib.rs

```diff
@@ -93,6 +99,12 @@ pub enum RuntimeMetadata {
 	/// Version 13 for runtime metadata, as raw encoded bytes.
 	#[cfg(not(feature = "v13"))]
 	V13(OpaqueMetadata),
+	/// Version 14 for runtime metadata.
+	#[cfg(feature = "v14")]
+	V14(v14::RuntimeMetadataV14),
+	/// Version 14 for runtime metadata, as raw encoded bytes.
+	#[cfg(not(feature = "v14"))]
+	V14(OpaqueMetadata),
```

Comment:
Would it work to name this variant `VNext` or `VTypeInfo` or something that saves us from doing this migration if/when substrate bumps to v14?

Reply:
It doesn't happen very often and I hope not to drag my PR out too much longer